src/Behat/Behat/Definition/Printer/ConsoleDefinitionPrinter.php

```diff
@@ -101,6 +104,11 @@ final protected function translateDefinition(Suite $suite, Definition $definitio
         return $this->translator->translateDefinition($suite, $definition);
     }
 
+    final protected function translateInfoText(string $infoText, array $parameters): string
```

Comment:
This function is used to translate the header info that is printed before the list of unused definitions